packages/app/src/index.ts

```diff
@@ -7,6 +9,7 @@ export { default as H5GroveProvider } from './providers/h5grove/H5GroveProvider'
 export { getFeedbackMailto } from './breadcrumbs/utils';
 export type { FeedbackContext } from './breadcrumbs/models';
 export type { ExportFormat, ExportURL } from './providers/models';
+export type GetExportURL = NonNullable<DataProviderApi['getExportURL']>;
```

Comment:
I'm declaring the type alias here directly to avoid cyclic imports.

Reply:
Thanks!  I was realizing I needed to define this function in the application code, and you're right I had to import a lot of types to try to build the function signature.